packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -114,6 +134,8 @@ export class AccountDeleteManager {
     });
     this.queueName = `projects/${tasksConfig.projectId}/locations/${tasksConfig.locationId}/queues/${tasksConfig.deleteAccounts.queueName}`;
     this.taskUrl = `${this.config.publicUrl}/v${this.config.apiVersion}${accountDeleteCloudTaskPath}`;
+    this.tasksEnabled = !!tasksConfig.credentials.keyFilename;
```

Comment:
I think SRE is planning to use Workload Identity so in stage and prod this is going to be false.

Reply:
oh I see that you have a `||` below so this could still work then.